components/org.apache.stratos.cli/src/main/java/org/apache/stratos/cli/RestCommandLineService.java

```diff
@@ -214,7 +215,7 @@ public void checkServerTrusted(java.security.cert.X509Certificate[] certs, Strin
                     if (responseCode == 200) {
                         return true;
                     } else {
-                        System.out.println("Invalid STRATOS_URL");
+                        System.out.println("Invalid value is set for STRATOS_URL");
```

Comment:
Create an overloaded "printError" method, which doesn't take an exception object. Having sysouts and another method to log errors makes it inconsistent on how you handle printing of errors.